house/house.go

```diff
@@ -143,6 +151,36 @@ func NewServer() HomeManager {
 	return server
 }
 
+func (s *Server) Devices(w http.ResponseWriter, req *http.Request) {
+	js, err := json.Marshal(append(iotDevices, houseDevices...))
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	w.WriteHeader(http.StatusOK)
+	w.Header().Set("Content-Type", "application/json")
+	w.Write(js)
+	return
+}
+
+func (s *Server) People(w http.ResponseWriter, req *http.Request) {
```

Comment:
warning: exported method Server.People should have comment or be unexported (golint)